### PR TITLE
Release 3 v2 apps (16)

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

The PR https://github.com/SumoLogic/sumologic-documentation/pull/3387, https://github.com/SumoLogic/sumologic-documentation/pull/3409, and https://github.com/SumoLogic/sumologic-documentation/pull/3427 are merged and created this new single PR.

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/web-servers/heroku.md [hunk 1]
```diff
@@ -196,6 +196,10 @@ These custom app **_sourceNames** will appear as values in the dashboard filter
 
 {@import ../../reuse/apps/app-install-v2.md}
 
+:::note
+While using the **Sumo Add-on**, the value **_sourceCategory=heroku** should be used while installing the app.
```
Comment (kimsauce) [inline docs/integrations/web-servers/heroku.md:200]:
```suggestion
While using the **Sumo Add-on**, the value `_sourceCategory=heroku` should be used while installing the app.
```

**Review by kimsauce — APPROVED** [review]

File: docs/integrations/microsoft-azure/windows-json-pci-compliance.md [hunk 2]
```diff
@@ -38,7 +37,7 @@ _sourceCategory=OS/Windows/Events ( "Audit Policy Change" or "System audit polic
 | count by msg_summary | sort by _count, msg_summary asc
 ```
 
-## Collecting Logs for the PCI Compliance for Windows JSON App
+## Collecting Logs for the PCI Compliance for Windows JSON app
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/windows-json-pci-compliance.md:40]:
```suggestion
## Collecting logs for the PCI Compliance for Windows JSON app
```

File: docs/integrations/security-threat-detection/zscaler-private-access.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 The Zscaler Private Access App collects logs from Zscaler using the Log Streaming Service (LSS) to populate pre-configured searches and Dashboards. The dashboards provide easy-to-access visual insights into user behaviors, security, connector status, and risk.
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/zscaler-private-access.md:12]:
```suggestion
The Zscaler Private Access app collects logs from Zscaler using the Log Streaming Service (LSS) to populate pre-configured searches and Dashboards. The dashboards provide easy-to-access visual insights into user behaviors, security, connector status, and risk.
```

File: docs/integrations/security-threat-detection/zscaler-private-access.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-## Log Types
+## Log types
 
 The Sumo Logic App for Zscaler Private Access uses LSS to send the following logs, as documented [here](https://help.zscaler.com/zpa/about-log-streaming-service):
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/zscaler-private-access.md:16]:
```suggestion
The Sumo Logic app for Zscaler Private Access uses LSS to send the following logs, as documented [here](https://help.zscaler.com/zpa/about-log-streaming-service):
```

File: docs/integrations/security-threat-detection/zscaler-private-access.md [hunk 5]
```diff
@@ -141,11 +127,12 @@ For **Syslog Token,** enter the token from the Sumo Logic[ Cloud Syslog Source](
 
 1. At this point, ZPA should start sending logs to Sumo Logic.
 
-## Installing the Zscaler Private Access app
+
+## Installing the Zscaler Private Access App
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/zscaler-private-access.md:131]:
```suggestion
## Installing the Zscaler Private Access app
```

File: docs/integrations/security-threat-detection/zscaler-private-access.md (hunk 4 above)
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/zscaler-private-access.md:16]:
```suggestion
The Sumo Logic app for Zscaler Private Access uses LSS to send the following logs, as documented [here](https://help.zscaler.com/zpa/about-log-streaming-service):
```

**Review by jpipkin1 — APPROVED** [review]
@JV0812, I approved the PR in anticipation of your making the small updates.

**Pull request closed without merging**